### DISCUSSION TITLE
Uncertainty with tag for hosts

### DISCUSSION THREAD

**Discussion by katruhanchik** [hazcod/crowdstrike-spotlight-slacker#7] [Q&A] [answered]
Please give an exhample how should the tag for hosts look like if their email is user.name@companydomain.com, because it isn't clear and here is what I receive:
![image](https://user-images.githubusercontent.com/37152641/123102735-090f7180-d43e-11eb-9ee4-6bedf4fed30b.png)

**Comment by hazcod** [comment]
Like mentioned on the readme:
```
Tag your Falcon hosts with email/user/company/com if their email is user@company.com.
```

**Reply by katruhanchik** [reply]
I tried some combinations and it resulted in stated error
e.g.:
email\john.doe\company\com
john.doe@company.com\john.doe\company\com
john.doe@company.com
john.doe\company.com

I tried them with and without quotation marks and with both slash and backslash

**Comment by hazcod** [comment, marked as answer]
`email/John/doe/company/com`
